easybuild/easyconfigs/c/CGAL/CGAL-4.5.1-intel-2015a-Python-2.7.9.eb

```diff
@@ -0,0 +1,30 @@
+name = 'CGAL'
+version = '4.5.1'
+
+homepage = 'http://www.cgal.org/'
+description = """The goal of the CGAL Open Source Project is to provide easy access to efficient 
+ and reliable geometric algorithms in the form of a C++ library."""
+
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+toolchainopts = {'strict': True}
+
+sources = [SOURCE_TAR_GZ]
+source_urls = ['http://fenicsproject.org/pub/software/contrib/']
+
+pythonversion = '2.7.9'
+versionsuffix = "-Python-%s" % pythonversion
+
+dependencies = [
+    ('GMP', '5.1.3'),
```

Comment:
All other dependencies are on this version of GMP.  There were issues with the Intel build, which have been resolved after deciding to go with 5.1.3 to avoid trouble.